primitives/consensus/common/src/import_queue/basic_queue.rs

```diff
@@ -57,14 +59,21 @@ impl<B: BlockT, Transaction: Send + 'static> BasicQueue<B, Transaction> {
 		justification_import: Option<BoxJustificationImport<B>>,
 		finality_proof_import: Option<BoxFinalityProofImport<B>>,
 		spawner: &impl sp_core::traits::SpawnBlocking,
+		prometheus_registry: Option<&Registry>,
 	) -> Self {
 		let (result_sender, result_port) = buffered_link::buffered_link();
+		let metrics = prometheus_registry.and_then(|r|
+			Metrics::register(r)
+			.map_err(|err| { log::warn!("Failed to register prometheus metrics: {}", err); })
```

Comment:
```suggestion
			.map_err(|err| { log::warn!("Failed to register Prometheus metrics: {}", err); })
```

Don't feel obliged to go through a whole new CI run cycle just for this change.